ios/RNLookback.m

```diff
@@ -82,4 +82,9 @@ - (dispatch_queue_t)methodQueue {
   [[Lookback sharedLookback] exitedView:view];
 }
 
+// check if lookback is recording
+RCT_REMAP_METHOD(getIsRecording, resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+    resolve([NSNumber numberWithBool:[[Lookback sharedLookback] isRecording]]);
```

Comment:
can you match the rest of the file's indentation? we use two spaces everywhere, thanks!